### PR TITLE
New Setup: Increment

### PR DESCRIPTION
**Pull request by simonbbaumgartner**
What is meant with the field "increment"? Is it "Totalbetrag der Amortisation über gesamte Laufzeit"?

We need more detailled information about increment. Could we please add the following fields after "increment"?

incrementType
description: The type of increment
type: string, enum: [direct, indirect]

incrementAmount
description: The amount of the periodic increment
type: see Amount

incrementStart
description: Start date of increment
type: see date

incrementPeriodicity
description: The periodicity the increment is paid type: string, enum [yearly, quarterly, monthly]

incrementAccountNr
description: the account number from which the increment is paid
type: string

### CODE REVIEW

File: mortgageAPI.yaml [hunk 1]
```diff
@@ -2394,6 +2394,22 @@ components:
           $ref: '#/components/schemas/AmountRange'
         increment:
           type: integer
+        amortizationType:
+          type: string
+          enum: [direct, indirect]
```
Comment (stjosh) [inline mortgageAPI.yaml:2399]: Pls add an example. Although this seems kind of overkill, it generates better / easier to understand examples in the Swagger UI when viewing the Spec.

File: mortgageAPI.yaml [hunk 2]
```diff
@@ -2394,6 +2394,22 @@ components:
...
+        amortizationAmount:
+          $ref: '#/components/schemas/Amount'
+          description: 'The amount of the periodic amortization'
+        amortizationStart:
+          $ref: '#/components/schemas/Date'
+          description: 'Start date of amortization'
+        amortizationPeriodicity:
+          type: string
+          enum: [yearly, quarterly, monthly]
+          description: 'The periodicity the amortization is paid'
```
Comment (stjosh) [inline mortgageAPI.yaml:2409]: Dito, pls add an example.

**Review by stjosh — CHANGES_REQUESTED** [review]

File: mortgageAPI.yaml (hunk 1 above)
Comment (simonbbaumgartner) [reply mortgageAPI.yaml:2399]: done, see below

File: mortgageAPI.yaml (hunk 2 above)
Comment (simonbbaumgartner) [reply mortgageAPI.yaml:2411]: done, see below

**Review by rolfwagner — APPROVED** [review]
ok, minor change. Will add even more examples.

**Review by rolfwagner — APPROVED** [review]
ok, minor change

**Pull request merged**